app/views/course_user_data/_fields.html.erb

```diff
@@ -7,7 +7,7 @@
 
   <% end %>
 
-  <%= f.text_field :nickname, help_text: "Anonymous nickname to display on the public scoreboards", placeholder: "droh" %>
+  <%= f.text_field :nickname, help_text: "Anonymous nickname to display on the public scoreboards (maxlength: 32)", placeholder: "droh", maxlength: 32 %>
```

Comment:
nit: add space in maxlength

Reply:
Fixed (PR description not updated)

Also made a small modification to the creation error message to make it consistent with the update error message.